v2/emailpassword/serverless/with-aws-lambda/auth-serverless.mdx

```diff
@@ -5,16 +5,27 @@ hide_title: true
 ---
```

Comment:
update the python env to use the latest python SDK version which was release. Make sure that all python code snippets added are correct. You may also need to Mangum as a python env dependency.

You can do code type checking:
```
CODE_TYPE_CHECK=python npm run start
```

Once you update the dependencies in pythonEnv, make sure to update the requirements.txt file:
```
pip freeze > requirements.txt
```

Reply:
Done.